lib/puppet/type/iis_app_pool.rb

```diff
@@ -0,0 +1,12 @@
+require 'puppet/parameter/boolean'
+require_relative '../../puppet_x/puppetlabs/iis/property/string'
+
+Puppet::Type.newtype(:iis_app_pool) do
```

Comment:
Pedantic naming discussion ahead, may not be useful right now:

Will we have` iis_app` or `iis_application` for MODULES-3050? I realize we have a convention for shortening names where possible, but I vote that `iis_applicationpool` or `iis_application_pool` is better at first glance to disambiguate than `iis_app_pool` next to `iis_app`.